recipes/imagl/all/conanfile.py

```diff
@@ -1,66 +1,81 @@
-from conans import ConanFile, CMake, tools, errors
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+required_conan_version = ">=1.32.0"
+
 
 class ImaglConan(ConanFile):
     name = "imagl"
-    license = "GPL-3"
+    license = "GPL-3.0-or-later"
     homepage = "https://github.com/Woazim/imaGL"
     url = "https://github.com/conan-io/conan-center-index"
-    description = "A lightweight library to load image for OpenGL application. Directly compatible with glTexImage* functions. Tested on linux, macOS and Windows with gcc, clang and msvc."
+    description = "A lightweight library to load image for OpenGL application."
     topics = ("opengl", "texture", "image")
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False]}
-    default_options = {"shared": False}
+    options = {"shared": [True, False], "fPIC": [True, False], "with_png": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "with_png": True}
     generators = "cmake"
-    requires = [("libpng/1.6.37")]
+    exports_sources = "CMakeLists.txt"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "9",
+            "Visual Studio": "16.5",
```

Comment:
```suggestion
            "Visual Studio": "16",
```

https://c3i.jfrog.io/c3i/misc/logs/pr/4225/4/imagl/0.1.0/summary.json

```json
{
        "reference": "imagl/0.1.0",
        "packageID": "INVALID",
        "settings": {
            "os": "Windows",
            "os_build": "Windows",
            "arch": "x86_64",
            "arch_build": "x86_64",
            "compiler": "Visual Studio",
            "compiler.version": "16",
            "build_type": "Debug",
            "compiler.runtime": "MDd"
        },
        "options": {
            "imagl:shared": "True"
        },
        "status": "DUPLICATED",
        "profile": null,
        "build": null,
        "test": null
    },
```

Reply:
Is it possible for you to commit this simple change, since the pull request is now approved? Or do I have to create a new pull request / issue?

Reply:
You'll need to make a new PR, this way CCI can generate all new packages